From 8d38721322bc47ce089f8d246513be610e22c187 Mon Sep 17 00:00:00 2001
From: ibizaman <ibizaman@tiserbox.com>
Date: Sun, 10 May 2026 08:58:10 +0200
Subject: [PATCH] never onboard

---
 backend/open_webui/main.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/backend/open_webui/main.py b/backend/open_webui/main.py
index e05497c..826af7b 100644
--- a/backend/open_webui/main.py
+++ b/backend/open_webui/main.py
@@ -2391,8 +2391,6 @@ async def get_app_config(request: Request):
             user = await Users.get_user_by_id(data['id'])
 
     onboarding = False
-    if user is None:
-        onboarding = not await Users.has_users()
 
     user_count = await Users.get_num_users() if app.state.LICENSE_METADATA else None
 
-- 
2.53.0
