From b09d31578f4091c81ec2352bd334a007a093d6df Mon Sep 17 00:00:00 2001
From: ibizaman <ibizapeanut@gmail.com>
Date: Fri, 31 Jan 2025 20:52:40 +0100
Subject: [PATCH] Make labels on login form point to correct inputs

---
 views/login.blade.php | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/views/login.blade.php b/views/login.blade.php
index 89538c3ea..ef708a657 100644
--- a/views/login.blade.php
+++ b/views/login.blade.php
@@ -15,7 +15,7 @@
 			novalidate>
 
 			<div class="form-group">
-				<label for="name">{{ $__t('Username') }}</label>
+				<label for="username">{{ $__t('Username') }}</label>
 				<input type="text"
 					class="form-control"
 					required
@@ -24,7 +24,7 @@ class="form-control"
 			</div>
 
 			<div class="form-group">
-				<label for="name">{{ $__t('Password') }}</label>
+				<label for="password">{{ $__t('Password') }}</label>
 				<input type="password"
 					class="form-control"
 					required
