--- a/pkgs/applications/networking/p2p/deluge/default.nix
+++ b/pkgs/applications/networking/p2p/deluge/default.nix
@@ -2,0 +3 @@
+  fetchpatch,
@@ -32,0 +34,14 @@
+      patches = [
+        (fetchpatch {
+          name = "deluge-remove-package-resources-dependency.patch";
+          url = "https://github.com/deluge-torrent/deluge/commit/18634913828900dedb836a81d96700cdadfb2b0a.patch";
+          excludes = [ "requirements.txt" ];
+          hash = "sha256-uqDoCumt6uljjlVY7K7qDuidD33oYO55cbNf3po0arY=";
+        })
+        (fetchpatch {
+          name = "deluge-replace-pyopenssl-certificate-generation.patch";
+          url = "https://github.com/deluge-torrent/deluge/commit/b763626ce88f9f9b82616de0a29087fde6d8828f.patch";
+          hash = "sha256-EtHcdZU1nCmLFrZiysoazRu9657NnuIbFdwRMhAWjI0=";
+        })
+      ];
+
@@ -40,0 +56,3 @@
+          cryptography
+          pkginfo
+          platformdirs
@@ -44 +61,0 @@
-          setuptools
